packages/orchestrator-ui-components/src/utils/strings.ts

```diff
@@ -8,3 +8,8 @@ export const removeSuffix = (value: string, splitChar = '?'): string => {
     if (value.length <= 1 || !value.includes(splitChar)) return value;
     return value.split(splitChar).slice(0, -1).join();
 };
+
+export const camelToHuman = (value: string): string => {
+    const result = value.replace(/([A-Z])/g, ' $1').trimStart();
```

Comment:
Are we certain certain that we don't encounter any numbers? `testTest1234` for example

Reply:
Numbers should be ok. We might need to check for resource types that start with a number; but I think that's against the python domain model naming (which is the source for the resource types)